crates/cli/src/common_args.rs

```diff
@@ -13,3 +14,10 @@ pub fn identity() -> Arg {
         .short('i')
         .help("The identity to use")
 }
+
+pub fn anonymous() -> Arg {
+    Arg::new("anon_identity")
+        .long("anonymous")
+        .action(SetTrue)
+        .help("Perform this action as an arbitrary newly-created user instead of as myself")
```

Comment:
I would update this to be a bit more exact, something like:
> Perform this action with an anonymous identity

specifically, using "user" here I think could be confusing